sanic/utils.py

```diff
@@ -105,6 +105,7 @@ def load_module_from_file_location(
             _mod_spec = spec_from_file_location(
                 name, location, *args, **kwargs
             )
+            assert _mod_spec is not None
```

Comment:
What's this for? Generally we try to avoid `assert` in application logic. If you are going to add it, can you an appropriate message.

Usually this would be preferable:

```python
if _mod_spec is None:
    raise SomeException
```

Here's why:

```python
$ python -O                                                                                                                                                               2 ↵
Python 3.9.4 (default, Apr 20 2021, 15:51:38) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> __debug__
False
>>> assert False
>>> 
```

---

With that said, I know there are a couple instances of it in the codebase. They should really be removed.

Reply:
It's a type assertion to keep mypy happy, the next line will crash anyway if _mod_spec is None

Reply:
Nice. Can you just add a quick comment before it so that we know the reason is just for `mypy`?

Reply:
```suggestion
            assert _mod_spec is not None # type assertion for mypy
```